shardingsphere-features/pom.xml

```diff
@@ -32,7 +32,7 @@
     <modules>
         <module>shardingsphere-sharding</module>
         <module>shardingsphere-replica</module>
-        <module>shardingsphere-master-slave</module>
+        <module>shardingsphere-primary-replica</module>
```

Comment:
The module name should be read-write-split

Reply:
Got it! 
If there are other places you can imagine to use other names, not just plain translate from master-slave to primary-replica, please let me know.

Reply:
> Hi, @flycoolman
> You should resolve the conficts first.
@kimmking Thanks for the suggestion. 
That will be my next action once the final decision about the module name is made.